lib/utils/oidc.js

```diff
@@ -96,17 +94,19 @@ async function oidc ({ packageName, registry, opts, config }) {
           const json = await response.json()
 
           if (!response.ok) {
-            throw new Error(`Failed to fetch id_token from GitHub: received an invalid response`)
+            log.silly('oidc', `Failed to fetch id_token from GitHub: received an invalid response`)
+            return undefined
           }
 
           if (!json.value) {
-            throw new Error(`Failed to fetch id_token from GitHub: missing value`)
+            log.silly('oidc', `Failed to fetch id_token from GitHub: missing value`)
```

Comment:
I think these service error conditions should be rare, and warrant verbose-level logging.

```suggestion
            log.verbose('oidc', `Failed to fetch id_token from GitHub: missing value`)
```